src/app/components/transaction/transaction-icon.tsx

```diff
@@ -19,7 +18,7 @@ export function TransactionIcon(props: { tx: StacksTx }) {
     case 'token_transfer':
       return <TokenTransferIcon tx={tx} />;
     case 'contract_call':
-      return <FunctionIcon size={token('sizes.xs')} />;
+      return <FunctionIcon size="xs" />;
```

Comment:
Any idea what / where this is? 

It seems to be a custom icon installed separately 
![Screenshot 2023-11-22 at 09 23 03](https://github.com/leather-wallet/extension/assets/2938440/69c46eed-bd87-4a3e-a9e5-ea1b0f915faf)

Maybe we can make our own icon doing this with a font